src/patternfly/components/NotificationDrawer/notification-drawer-task-group-toggle-action.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-notification-drawer__task-group-toggle-action{{#if notification-drawer-task-group-toggle-action--modifier}} {{notification-drawer-task-group-toggle-action--modifier}}{{/if}}"
+  {{#if notification-drawer-task-group-toggle-action--attribute}}
+    {{{notification-drawer-task-group-toggle-action--attribute}}}
+  {{/if}}>
+  <a class="pf-c-button pf-m-inline pf-m-link">{{> @partial-block}}</a>
```

Comment:
Looks like "action" gets passed to these - curious why it's an `<a>`? Also if we keep this as the butotn component, can we use the `{{#> button}}` hbs to generate this element?

Reply:
@mcoker This is related to the issue above of the action link nested inside a button. I think we could use `{{#> button-link }}` but it wasn't creating the underline on hover, which I thought was better to have since there's also an action on clicking the entire toggle. Anyway we need to resolve how to handle the link inside the toggle first.